src/System.CommandLine/Parsing/ArgumentResult.cs

```diff
@@ -20,9 +20,11 @@ public class ArgumentResult : SymbolResult
 
         public IArgument Argument { get; }
 
+        public bool IsImplicit => PassedOnTokens == null;
```

Comment:
I'm not sure this is correct. `IsImplicit` means something a bit different in the existing `OptionResult.IsImplicit`, which is that the token was generated by the parser as a sort of placeholder for a default value. 

At the least, the `PassedOnTokens == null` case looks like it might only accidentally correlate with the presence of a default value. This collection is _intended_ to capture the overflow when a custom parser doesn't want to consume all of the available tokens.